feat: remove background, add hover, delete background parameter to icon

cdrage opened the pull request:
feat: remove background, add hover, delete background parameter to icon

### What does this PR do?

* Removes the background on list and details for a "hover" circle
* Delete the background parameter for icons

Note: Reasoning for deleting background parameter: No longer needed as
no longer adding a background. Tailwind also does not show the hover
correctly as a parameter (ex. class="{backgroundColor} m-2") with hover.

### Screenshot/screencast of this PR

![Screenshot 2022-12-06 at 12 18 59 PM](https://user-images.githubusercontent.com/6422176/205979121-b49a0ddf-03bb-4a4d-8d12-bc8d0bd52387.png)
![Screenshot 2022-12-06 at 12 19 07 PM](https://user-images.githubusercontent.com/6422176/205979123-ea420b17-e834-4029-82eb-22949889eee9.png)


### What issues does this PR fix or reference?

<!-- Please include any related issue from Podman Desktop repository (or from another issue tracker).
-->

N/A

### How to test this PR?

<!-- Please explain steps to reproduce -->

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

benoitf commented:
could you please only enhance the list without the details ?

cdrage commented:
@benoitf Were retrieving the class from the same file. 

Changing one unfortunately affects the other until UI/UX finalized what it looks like on the detailed screen.

We could add an if statement as well as another file similar to how we did `FlatMenu.svelte` and `DropdownMenu.svelte`?

Could we do this for now?
![Screenshot 2022-12-08 at 11 24 47 AM](https://user-images.githubusercontent.com/6422176/206505475-3447e63b-b90d-4572-805c-c0c21d4c8ef4.png)
![Screenshot 2022-12-08 at 11 24 32 AM](https://user-images.githubusercontent.com/6422176/206505482-7c4a80f0-d858-4fd2-baa0-c1c8b59ea9f5.png)

benoitf commented:
I think we can provide flags to keep existing layout for one case and not the other one. I would not introduce a rectangle or anything, just keeping the previous rendering.

cdrage commented:
Updated!

See below screenshots.

I've also updated it across all other parts too (volumes and images).
![Screenshot 2022-12-08 at 3 05 04 PM](https://user-images.githubusercontent.com/6422176/206556825-9fa1785d-5369-4aaa-9404-fd3e9000ebad.png)
![Screenshot 2022-12-08 at 3 04 59 PM](https://user-images.githubusercontent.com/6422176/206556826-3a1fef4b-9c63-4488-aad1-6fb7ffa2a332.png)
![Screenshot 2022-12-08 at 3 04 53 PM](https://user-images.githubusercontent.com/6422176/206556829-7a53caad-4544-4c66-ac99-24778c31bb99.png)

benoitf reviewed (approved):
tested, works as expected 🎉 

https://user-images.githubusercontent.com/436777/206649453-a8394ce1-12fe-4fe6-b8a3-e33f29af41d3.mp4
